scripted-tests/run/java-io-file-1/Utils.scala

```diff
@@ -0,0 +1,18 @@
+import java.util.Locale
+
+object Utils {
+  val isWindows = System
+    .getProperty("os.name", "unknown")
+    .toLowerCase()
```

Comment:
```suggestion
    .toLowerCase(Locale.ROOT)
```
with the appropriate `import`.